jwst/lib/siafdb.py

```diff
@@ -204,10 +204,10 @@ def get_wcs(self, aperture, useafter):
         aperture = siaf[aperture.upper()]
 
         # Fill out the Siaf
-        verticies = tuple(getattr(aperture, key) for key in SIAF_VERTICIES)
+        vertices = tuple(getattr(aperture, key) for key in SIAF_VERTICES)
```

Comment:
as above, do not change SIAF_VERTICIES.

Reply:
Note: I've changed `vertices` but have left `SIAF_VERTICIES` unchanged.